src/config.rs

```diff
@@ -14,7 +14,17 @@ macro_rules! pub_struct {
 
 pub_struct!(Server { port: u16, });
 
-pub_struct!(Database {
+pub_struct!(Databases {
+    starknetid: StarknetidDb,
+    sales: SalesDb,
+});
+
+pub_struct!(StarknetidDb {
```

Comment:
Maybe we could have a common struct `Database` as the data is the same for both db

Reply:
good idea